app/models/concerns/episode_media.rb

```diff
@@ -31,8 +31,8 @@ def complete_media?
     complete_media.any?
   end
 
-  def no_media?
-    medium.nil? && segment_count.nil? && !media?
+  def media?
+    medium.present? || segment_count.present? || media.present?
```

Comment:
The reason the `<enclosure>` wasn't appearing in the RSS - is because there was a difference in the `no_media?` and `media?` logic.

I consolidated to a single method here.  Episodes are always expected to have media if they have a medium, segment_count, or media (contents).